code/game/mecha/equipment/tools/work_tools.dm

```diff
@@ -43,7 +43,7 @@
 					O.forceMove(chassis)
 					O.anchored = FALSE
 					occupant_message("<span class='notice'>[target] successfully loaded.</span>")
-					log_message("Loaded [O]. Cargo compartment capacity: [cargo_holder.cargo_capacity - cargo_holder.cargo.len]")
+					log_message("Loaded [O]. Cargo compartment capacity: [cargo_holder.cargo_capacity - cargo_holder.cargo.len]", LOG_MECHA)
```

Comment:
you can revert all of these LOG_MECHA changes, they're the result of a small rework of how logs work on tg